src/arch/xtensa/configs/rembrandt_defconfig

```diff
@@ -33,3 +33,4 @@ CONFIG_COMP_TDFB=n
 ##CONFIG_COMP_MUX=n
 CONFIG_COMP_SEL=n
 CONFIG_COMP_MIXER=n
+CONFIG_DISABLE_DESCRIPTOR_SPLIT=n
```

Comment:
these aren't needed, since the option is disabled by default, are they?

Reply:
Correct not needed, will remove!